### ISSUE TITLE
Case Data Fields Spacing, List Treatment, Design 

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#817]
FOR P2 - Part 2 of #775 (See #816 for Part 1)

1. All multi-select fields (eg. General Issues, Specific Topics, Purpose, etc.) to be listed on separate lines.

2. Get rid of all text in brackets, for example "Governance & Political Institutions (e.g. Constitutions, Legal Systems, Electoral Systems)" would become "Governance & Political Institutions".

3. Indent any 'overflow text' ie. anything that doesn't fit on one line. For example:

Governance & Political
--> Institutions
Law Enforcement, Criminal
--> Justice & Corrections

4. Location data does not need to display coordinates.

5. All values should have the same spelling/grammar/ordering as the data forms - see this document: https://docs.google.com/document/d/1sovR-RCilKwWN2XKxQH7xVo8XDEnh6BSgz7JfNqFy4g/edit#

6. 'True' should be 'Yes', 'False should be 'No'

**Comment by jesicarson** [comment]
Amber approved items 1 and 3 above, I'll provide a mockup with spacing specs soon. 
2 - if its easy for dethe to make this happen, ok. if not, maybe not worth it.
4 - sounds good.
5 - yes, and admins will be able to edit grammar etc soon! yay!
6 - sounds good.

**Comment by scottofletcher** [comment]
@jesicarson cool beans! yeah the bracket thing isn't a necessity - your work on 1 and 3 might make things easy enough to read without getting rid of the brackets. I'm just worried about some of the method values which have really long explainers in brackets...
RE 5: I'm actually excited for this!! it's just going to be grammar though, right? not adding missing values?

**Comment by jesicarson** [comment]
Good question.. pretty sure you will be able to add values tho. Working on the ui and will confirm with Dethe what is possible and keep u posted. Would be best of admins can also add/ delete values.

**Comment by scottofletcher** [comment]
@jesicarson there have also been some problems around selecting/deselecting values that were ported from .net (see participedia/usersnaps#207 participedia/usersnaps#203)

**Comment by jesicarson** [comment]
Saw those snaps, are you saying they need to be prioritized?

Also regarding values with really long explainers in brackets - BEST course of action would be to shorten them as much as possible.

**Comment by scottofletcher** [comment]
no, they don't need to be prioritized, I was just wondering if that would be possible for admins to do since they are values. 

RE long explainers: we did, any shorter and I don't think people would understand. I think I'm probably overthinking things - let's just see what happens when 1 and 3 are completed

**Comment by jesicarson** [comment]
ai file [here](https://drive.google.com/open?id=1fjo-m2NqSwIllXedWyYb8vlKpY8wp-XX)

most up to date design for cases - sidebar indentation shown. 

![case styleguide draft june 26 2018](https://user-images.githubusercontent.com/13560530/41933626-4b893400-7939-11e8-8feb-1328519ec686.jpg)
